tests/unit/drivers/test_vector_search_driver.py

```diff
@@ -7,6 +7,7 @@
 from jina.drivers.search import VectorSearchDriver
 from jina.executors.indexers import BaseVectorIndexer
 
+# TODO: Add tests for sparse vectors search driver.
```

Comment:
just a reminder

Reply:
@bwanglzu I have done changes to avoid having extra Driver classes, so that it is easier for the user to use.

Still the VectorIndexDriver and VectorSearchDriver need to be unit tested